### PR TITLE
Pull in Microprofile Fault Tolerance 2.0 API

### LINKED ISSUE #6234: FT 2.0: Consume API bundle
Consume the FT 2.0 API bundle and include it in liberty

### PR DESCRIPTION
**Pull request by Azquelt**
In addtion, as it depends explicitly on the CDI 2.0 API, remove the FT2.0-CDI1.2
autofeature as that configuration is not supported

Fixes #6234

### CODE REVIEW

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
Build didn't complete, but it did pass the tests for this module so I'm hopeful that a new build will pass.

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
#build

File: dev/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.2.0/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/bin/
```
Comment (tevans78) [inline dev/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.2.0/.gitignore]: .ignore file not needed

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-2.0/com.ibm.websphere.appserver.mpFaultTolerance-2.0.feature [hunk 2]
```diff
@@ -8,7 +8,7 @@ IBM-API-Package: org.eclipse.microprofile.faulttolerance.exceptions;  type="stab
                  org.eclipse.microprofile.faulttolerance;  type="stable"
 IBM-ShortName: mpFaultTolerance-2.0
 Subsystem-Name: MicroProfile Fault Tolerance 2.0
--features=com.ibm.websphere.appserver.org.eclipse.microprofile.faulttolerance-1.1, \
+-features=com.ibm.websphere.appserver.org.eclipse.microprofile.faulttolerance-2.0, \
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-2.0/com.ibm.websphere.appserver.mpFaultTolerance-2.0.feature]: should there be a hard dependency on cdi-2.0?

**Comment by Azquelt** [comment]
Errors in the build do not appear to be related to this change.

Fixes #6234

**Pull request merged**